src/main/java/uk/gov/hmcts/reform/cpo/data/CasePaymentOrderAuditEntity.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.cpo.data;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.UUID;
+
+@Data
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "case_payment_orders_audit")
+public class CasePaymentOrderAuditEntity implements Serializable {
+
+    private static final long serialVersionUID = 3201769457154562891L;
+
+    @NonNull
+    @Id
```

Comment:
UUID id; will not let any set of null values or invalid format , may we be we don't need @NonNull